### ISSUE TITLE
Open-vocabulary segmentation Demo

### ISSUE THREAD

**Issue by yashbhalgat** [FoundationVision/GLEE#30]
Hi @wjf5203, thank you for this interesting work!

Can you please provide a "demo" script for **open-vocabulary** instance segmentation on videos? Currently, I only see a `TEST.md` file that describes evaluating on existing datasets. 

I also find the code a bit hard to follow to be able to write this open-vocabulary segmentation on my own. So, it would be very helpful if you can provide such a script. 

Thank you! :)
- Yash

**Comment by wassimea** [comment]
+1

**Comment by xiao10ma** [comment]
Have you solved the problem?
